documentation/design-docs/dotnet-tools.md

```diff
@@ -307,7 +307,7 @@ CONVERT
 
     dotnet-trace convert [-h|--help]
                          [-o|--output <output_file_path>]
-                         [--to-speedscope]
+                         -f|--format <format>
```

Comment:
The tools team has recommended we use single character abbreviations sparingly and only for very common operations. -h for help and -o for output are considered common, but -f for format probably would not. I'd suggest eliminating the '-f' short-form